contrib/run_task.sh

```diff
@@ -83,15 +131,19 @@ do_test() {
     $cargo build
     $cargo test
 
-    for example in $EXAMPLES; do
-	name="$(echo "$example" | cut -d ':' -f 1)"
-	features="$(echo "$example" | cut -d ':' -f 2)"
-	$cargo run --example "$name" --features="$features"
+    # Split the EXAMPLES string into an array based on spaces.
+    IFS=' ' read -r -a examples_array <<< "$EXAMPLES"
```

Comment:
In d457ed8d54d72fdb7e09b69d38527f864f022a75:

If we are going to do this hideous thing we might as well do it in `test_vars.sh`. Though it would be better to just define the array directly rather than using a string variable and then `read -r -a`ing it. (Alternately, the original code was perfectly fine.)

Reply:
The examples were not being run for some reason (IIRC), I only noticed when chatGPT suggested this and I tried it and got a bunch of new errors/warnings. I' not certain because the original shell looks ok to me to.

Reply:
I must have been mad, I re-tried the original and it seems to work just fine.